Cargo.toml

```diff
@@ -13,21 +13,18 @@ doctest = false
 [features]
 unstable = [
     "string_cache/unstable",
-    "tendril/unstable",
     "rc/unstable",
     "html5ever/unstable",
     "selectors/unstable",
 ]
-with-hyper = ["hyper"]
```

Comment:
I ended up making that `traits` module anyway. Do you think `from_http` would be better in an extension trait in kuchki?